Clarify Cordova plugin testing info

cookieguru opened the pull request:
Until I started reading responses in various issues it wasn't clear to me that you need to build the app in order to test any Cordova plugins that aren't already built in.  This PR is simply to update the docs to make that fact a little more explicit.

On a related note, is it technically possible to pull in plugins at runtime from within the developer app?  In other words, is this something that could be possible in a future version?  One of the greatest features of Tabris.js is the ability to rapidly see code changes on a device, and if an app relies on a Cordova plugin it just makes the process longer to have to build it every time you need to fix a typo.

jonek commented:
> On a related note, is it technically possible to pull in plugins at runtime from within the developer app?
> In other words, is this something that could be possible in a future version? One of the greatest
> features of Tabris.js is the ability to rapidly see code changes on a device, and if an app relies on a
> Cordova plugin it just makes the process longer to have to build it every time you need to fix a typo.

Just to clarify some things:
1. The [Tabris.js Developer App]() already contains some Cordova Plugins. You can see them by clicking on the 'plug' icon in the toolbar.
2. If you build your own app you can have any Cordova plugins in there while still having access to the developer console. The developer console (bezzle swipe from the right) will provide a URL input field in future version. That will enable you to access the JS code from your development machine while having your own set of Cordova plugins. The Cordova plugins are a fixed part of the installed app, while the JS code can be loaded from anywhere.
3. You can't "load" Cordova plugins at runtime like tthe JS code. Cordova plugins typically contain native parts (in Java for Android and in objectveC for iOS) next to their JS part. Of course this native parts have to be build (compiled) into the app and can't loaded at runtime like a JavaScript file.

cookieguru commented:
1. I also updated this list to reflect all the plugins included in 1.1.
2. Excellent!  I changed the wording in the doc to reflect this.  I didn't know this was possible.
3. This answers my question, thank you.

tbuschto commented:
I submitted these changes to Gerrit.

tbuschto commented:
The change was submitted with some further revision by myself.
